engine/rowset/src/main/java/io/deephaven/engine/rowset/impl/sortedranges/SortedRanges.java

```diff
@@ -460,6 +460,7 @@ private static final class Iterator implements RowSet.Iterator {
         private int nextRangeIdx = 0;
         private long rangeCurr = -1;
         private long rangeEnd = -1;
+        private long cachedNextValue;
```

Comment:
I would call this field `nextValue` instead of `cachedNextValue`, because I think is more in line with the rest of the naming of fields we do in Iterators around index.